_posts/2021-02-16-cockpit-238.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 238
+author: kkoukiou
+date: 2021-02-17
+tags: cockpit linux
+slug: cockpit-238
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 238.
+
+### Updates: Show the list of services and applications that are outdated
+
+Previously, we required a system reboot after every update. We now use [Tracer](http://tracer-package.com/)
+to display the list of services and applications that are outdated and need to be restarted or require a system reboot.
+
+If there are not such services not system reboot will be suggested.
+
+![Update no restart](/images/update-no-restart.png)
+
+If there are such services, the user will be asked to either to restart these or schedule a system reboot.
+
+![Update with restart](/images/update-with-restart.png)
+
+### Web server: Do not change permissions of administrator provided certificates
+
+Cockpit's web server supports and encourages [using your own TLS certificate/key](https://cockpit-project.org/guide/latest/https.html#https-certificates) in `/etc/cockpit/ws-certs.d/`.
+Previously, starting cockpit would adjust the file permissions and SELinux context of administrator provided
+certificates.  But commonly such files are shared with other services, in which case the permission changes could render
+the file inaccessible for these other services.
+
+Now cockpit only adjusts the permissions of certificate files which it creates and manages itself, in particular
+`0-self-signed.cert` and `10-ipa.cert`. If you provide your own certificate, you must make sure yourself that these are
+readable by the `cockpit-ws` user or group.
+
+### System: The performance metrics page now shows up to 16 cores ordered by their load
```

Comment:
```suggestion
### System: Performance page orders 16 or fewer cores by load
```

Reply:
I find the replacement sentence confusing. It's not really the function of the page to order the CPU cores by load. Plus, that features s not about the ordering so much, but about showing these.

Reply:
Ah, then that means I misunderstood your heading.

Reply:
```suggestion
### System: Performance page shows up to 16 cores, ordered by load
```

But, really, the order doesn't matter, does it?

```suggestion
### System: Performance page shows up to 16 cores
```